ui_framework/components/icon/delete_icon.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { KuiIcon } from './kui_icon';
+
+export function DeleteIcon() {
+  return <KuiIcon className="fa-trash"/>;
```

Comment:
The are case-sensitivity issues on OS-es, but as long as you import correctly it isn't a problem (e.g. if you name the file `Test.js` but import `test.js` it will break depending on OS). I expect there's an eslint to handle this too? /cc @spalger

Many +1s from me on CamelCasing.js